lib/src/rules/unnecessary_nullable_for_final_variable_declarations.dart

```diff
@@ -63,6 +65,19 @@ class _Visitor extends SimpleAstVisitor<void> {
   final LinterContext context;
   _Visitor(this.rule, this.context);
 
+  void check(AstNode? element) {
```

Comment:
I think this is only called with non-nullable `DartPattern`s.

Reply:
Good catch!